### PR TITLE
Fix hard-coded errors

### PR DESCRIPTION
**Pull request by Aritra8438**

### CODE REVIEW

File: api/templates/api-documentation.html [hunk 1]
```diff
@@ -176,8 +181,9 @@ <h3>Welcome to the API-documentation Page</h3>
           <br />
           <label
             ><strong>Description</strong>: <br />This API returns a table of
-            population, gdp_per_capita or forest area <br/> (in accordance with
-             the query type)  with countries and years as columns. <br /><br />
+            population, gdp_per_capita or forest area <br />
+            (in accordance with the query type) with countries and years as
```
Comment (raisachatterjee) [inline api/templates/api-documentation.html:185]: Is this because of prettify ?

**Review by raisachatterjee — APPROVED** [review]

File: .pylintrc [hunk 2]
```diff
@@ -1,4 +1,4 @@
 [MASTER]
 
 [MESSAGES CONTROL]
-disable = C0114, C0115, C0116, R0903
+disable = C0114, C0115, C0116, R0903
```
Comment (Aritra8438) [inline .pylintrc:4]: What has changed here?

If unwanted remove it and then merge.

Comment (raisachatterjee) [reply .pylintrc:4]: Nothing was changed. I added W0611 and then removed.

Comment (Aritra8438) [reply .pylintrc:4]: Then why git diff is showing changes?
Maybe you have added a new line? Can you remove it?

Comment (raisachatterjee) [reply .pylintrc:4]: No, I didn't add a new line.

File: api/templates/api-documentation.html (hunk 1 above)
Comment (Aritra8438) [reply api/templates/api-documentation.html:185]: Yes, `prettier`

**Pull request merged**